diff --git a/debug/gdbserver.py b/debug/gdbserver.py
index af8ddcf..f4b65b0 100755
--- a/debug/gdbserver.py
+++ b/debug/gdbserver.py
@@ -871,7 +871,7 @@ class MemorySampleSingle(MemorySampleTest):
         self.gdb.command(f"monitor riscv memory_sample 0 0x{addr:x} {sizeof_j}")
 
         raw_samples = self.collect_samples()
-        self.check_incrementing_samples(raw_samples, addr, tolerance=0x500000)
+        self.check_incrementing_samples(raw_samples, addr, tolerance=0x700000)
 
         # Buffer should have been emptied by dumping.
         raw_samples = self.gdb.command("monitor riscv dump_sample_buf", ops=5)
@@ -891,7 +891,7 @@ class MemorySampleMixed(MemorySampleTest):
 
         raw_samples = self.collect_samples()
         self.check_incrementing_samples(raw_samples, addr["j"],
-                                        tolerance=0x500000)
+                                        tolerance=0x700000)
         self.check_samples_equal(raw_samples, addr["i32"], 0xdeadbeef)
         if self.hart.xlen >= 64:
             self.check_samples_equal(raw_samples, addr["i64"],
