From 830a50989dba89d632c6639cad1be720007c3445 Mon Sep 17 00:00:00 2001
From: Parshintsev Anatoly <anatoly.parshintsev@syntacore.com>
Date: Mon, 29 Jan 2024 23:42:01 +0300
Subject: fixup setting of remotetimeout

---
 debug/testlib.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debug/testlib.py b/debug/testlib.py
index 7f28186..8abcf52 100644
--- a/debug/testlib.py
+++ b/debug/testlib.py
@@ -743,7 +743,6 @@ class Gdb:
             # Force consistency.
             self.command("set print entry-values no", reset_delays=None)
             self.command(f"set remotetimeout {self.timeout}", reset_delays=None)
-            self.command(f"set remotetimeout {self.target.timeout_sec}")
         self.active_child = self.children[0]
 
     def connect(self):
-- 
2.40.1

