diff --git a/src/server/server.c b/src/server/server.c
index 4e970fa8f..43bd9fb2e 100644
--- a/src/server/server.c
+++ b/src/server/server.c
@@ -725,6 +725,10 @@ void server_free(void)

 void exit_on_signal(int sig)
 {
+	/* dump coverage before being killed by the signal
+	 * (otherwise gcov's *.gcda files would not be created) */
+	void __gcov_flush(void);
+	__gcov_flush();
 #ifndef _WIN32
 	/* bring back default system handler and kill yourself */
 	signal(sig, SIG_DFL);
