diff -purN libusb-win32-src-1.2.6.0/src/install.c libusb-win32-src-1.2.6.0-mingw-w64/src/install.c
--- libusb-win32-src-1.2.6.0/src/install.c	2011-10-28 14:34:08.000000000 +0200
+++ libusb-win32-src-1.2.6.0-mingw-w64/src/install.c	2013-05-30 11:28:54.000000000 +0200
@@ -31,14 +31,8 @@
 #include <conio.h>
 #include <ctype.h>
 
-#ifdef __GNUC__
-#if  defined(_WIN64)
-#include <cfgmgr32.h>
-#else
-#include <ddk/cfgmgr32.h>
-#endif
-#else
 #include <cfgmgr32.h>
+#ifndef __GNUC__
 #define strlwr(p) _strlwr(p)
 #endif
 
diff -purN libusb-win32-src-1.2.6.0/src/install_filter_win.c libusb-win32-src-1.2.6.0-mingw-w64/src/install_filter_win.c
--- libusb-win32-src-1.2.6.0/src/install_filter_win.c	2010-09-22 22:35:52.000000000 +0200
+++ libusb-win32-src-1.2.6.0-mingw-w64/src/install_filter_win.c	2013-05-30 11:39:19.000000000 +0200
@@ -21,15 +21,9 @@
 
 #include <windows.h>
 
-#ifdef __GNUC__
-	#if  defined(_WIN64)
-		#include <cfgmgr32.h>
-	#else
-		#include <ddk/cfgmgr32.h>
-	#endif
-#else
-	#include <cfgmgr32.h>
-	#define strlwr(p) _strlwr(p)
+#include <cfgmgr32.h>
+#ifndef __GNUC__
+#define strlwr(p) _strlwr(p)
 #endif
 
 #include <commdlg.h>
diff -purN libusb-win32-src-1.2.6.0/src/registry.c libusb-win32-src-1.2.6.0-mingw-w64/src/registry.c
--- libusb-win32-src-1.2.6.0/src/registry.c	2010-09-23 17:40:50.000000000 +0200
+++ libusb-win32-src-1.2.6.0-mingw-w64/src/registry.c	2013-05-30 11:29:22.000000000 +0200
@@ -24,14 +24,8 @@
 #include <stdlib.h>
 #include <string.h>
 
-#ifdef __GNUC__
-#if  defined(_WIN64)
-#include <cfgmgr32.h>
-#else
-#include <ddk/cfgmgr32.h>
-#endif
-#else
 #include <cfgmgr32.h>
+#ifndef __GNUC__
 #define strlwr(p) _strlwr(p)
 #endif
 
